homeassistant/components/plex/config_flow.py

```diff
@@ -50,7 +51,7 @@ def configured_servers(hass):
 class PlexFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a Plex config flow."""
 
-    VERSION = 1
+    VERSION = 2
```

Comment:
Do we need to migrate existing entries to also store the client id?

Reply:
Unfortunately not possible. They will need to be recreated if one is needed in the future.

Reply:
We can't ask for the client id from the auth client after login in?

Reply:
No, it's up to the client to save it. The future change is part of a security hardening where a token created by this auth flow will only be usable if the original client identifier used to request the token is also provided. As of today a token can be used by itself to login.

Reply:
I'm not seeing a way to retrieve it after it's been discarded. The config entry version is purely informational for a quicker way to see if it was created by a handler that saves the client identifier. But it also assumes there's a migration handler from one version to another. Perhaps it's not worth a config entry version bump?

Reply:
Ok. I'm not sure about the version bump.

Reply:
Let's revert and keep at 1 for now since there's no existing features that will break and no known timeline for a change. Can make an announcement that integrations need to be recreated if that is ever necessary.

Maybe if a future feature depends on this a check can be added then which will alert the user.